app/main.py

```diff
@@ -1,3 +1,65 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union["Distance", int, float]) -> "Distance":
```

Comment:
```suggestion
    def __add__(self, other: Union[Distance, int, float]) -> "Distance":
```
please fix everywhere